frontend/simple/views/Join.vue

```diff
@@ -0,0 +1,164 @@
+          </div>
+          <div class="column is-one-quarter">
+            <table class="table is-bordered is-striped is-narrow">
+            <thead>
+            <tr>
+              <th><i18n>Group Members</i18n></th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr v-for="member in contract.members" class="member">
+              <td>
+                <div class="media">
+                  <div class="media-left">
+                    <p class="image is-64x64">
+                      <img src="http://bulma.io/images/placeholders/128x128.png">
+                    </p>
+                  </div>
+                  <div class="media-content">
+                    <strong>{{member}}</strong>
+                  </div>
+                </div>
+              </td>
+            </tr>
+            </tbody>
+          </table>
+          </div>
+        </div>
+      </div>
+    </section>
+</template>
+<script>
+import * as Events from '../../../shared/events'
+import backend from '../js/backend/'
+import * as db from '../js/database'
+import {latestContractState} from '../js/state'
+export default {
+  name: 'Join',
+  mounted () {
+    this.fetchData()
+  },
+  methods: {
+    fetchData: async function () {
+      let state = await latestContractState(this.$route.query.groupId)
+      console.log(state)
+      if (!state.invitees.find(invitee => invitee === this.$store.state.loggedIn)) {
+        console.log(new Error('Invalid Invitation'))
+        this.$router.push({path: '/mailbox'})
+      }
+      this.contract = state
+    },
+    accept: async function () {
+      try {
+        this.errorMsg = null
+        await backend.subscribe(this.$route.query.groupId)
+        await this.$store.dispatch('syncContractWithServer', this.$route.query.groupId)
+        let latest = await db.recentHash(this.$route.query.groupId)
+        let acceptance = new Events.AcceptInvitation({ username: this.$store.state.loggedIn, inviteHash: this.inviteHash, acceptanceDate: new Date() }, latest)
+        this.$store.commit('setCurrentGroupId', this.$route.query.groupId)
+        await backend.publishLogEntry(this.$route.query.groupId, acceptance)
+        this.$store.dispatch('deleteMail', this.$route.query.inviteHash)
+      } catch (ex) {
+        console.log(ex)
+        this.errorMsg = 'Failed to Accept Invite'
+      }
+    },
+    decline: async function () {
+      try {
+        this.errorMsg = null
+        let latest = await backend.latestHash(this.$route.query.groupId)
+        let declination = new Events.DeclineInvitation({ username: this.$store.state.loggedIn, inviteHash: this.inviteHash, declinedDate: new Date() }, latest)
+        await backend.publishLogEntry(this.$route.query.groupId, declination)
+        this.$store.dispatch('deleteMail', this.$route.query.inviteHash)
+        this.$router.push({path: '/mailbox'})
+      } catch (ex) {
+        console.log(ex)
+        this.errorMsg = 'Failed to Decline Invite'
```

Comment:
All user-facing strings, including strings defined in JS like here, need to be i18n wrapped. So this needs to be wrapped in the `L` function (or whatever it was).